### DISCUSSION TITLE
Remove the usage of < and > in world mods selection menu

### DISCUSSION THREAD

**Discussion by ghost** [cataclysmbn/Cataclysm-BN#3281] [General]
New players don't even understand how this menu works and these hotkeys are some of the worst we could expect a new player to understand. I have no idea of how to make this UI better.
![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/112293514/835debb4-3c7b-4eea-adb1-ce60baa5c30d)
![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/112293514/063f140b-6f77-4175-9ffd-b3a24fbc06c7)

**Comment by RobbieNeko** [comment]
Especially with how tab is already reasonably taken up by the top sections (World Mods, World Options, Finalize World), and left and right are already used to switch between the inactive and active mods.

**Comment by Relgar** [comment]
We can break up the tabs into separate pages with a Previous / Next bottom at each page. It'll be longer, but more natural for newer players.
